notification-service/src/main/java/org/hypertrace/alert/engine/notification/service/NotificationChannelsReader.java

```diff
@@ -38,6 +39,8 @@
       throw new IOException("File should contain an array of notification rules");
     }
 
+    LOGGER.info("Reading notification rules {}", jsonNode.toPrettyString());
```

Comment:
lets make it to `debug`.

Reply:
this is one time and helpful in knowing what were the rules

Reply:
ok, but if it's too much logging, can be wrapped inside debug.